TFT/src/User/API/Printing.c

```diff
@@ -584,10 +592,14 @@ bool printPause(bool isPause, PAUSE_TYPE pauseType)
           if (isCoorRelative == true)    mustStoreCmd("G90\n");
           if (isExtrudeRelative == true) mustStoreCmd("M82\n");
 
-          if (heatGetCurrentTemp(heatGetCurrentHotend()) > infoSettings.min_ext_temp)
-          {
+          if (extrusionDuringPause == true)  // check if extrusion done during Print -> Pause
+          { // no purge
+            extrusionDuringPause = false;
+          }
+          else if (heatGetCurrentTemp(heatGetCurrentHotend()) > infoSettings.min_ext_temp)
+          { // purge
             mustStoreCmd("G1 E%.5f F%d\n", tmp.axis[E_AXIS] - infoSettings.pause_retract_len + infoSettings.resume_purge_len,
-                         infoSettings.pause_feedrate[FEEDRATE_E]);
+                        infoSettings.pause_feedrate[FEEDRATE_E]);
```

Comment:
as far as I see in all the code, the code is aligned after "(", so the previous code was correct

Reply:
I didn't touch it, must be my VSCode acting on its own... 
It's fixed now..